test/Microsoft.Extensions.Logging.Test/LoggerFilterTest.cs

```diff
@@ -1,9 +1,9 @@
 ﻿// Copyright (c) .NET Foundation. All rights reserved.
 // Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
 
-using System;
 using System.Linq;
 using Microsoft.Extensions.Logging.Test;
+using Microsoft.Extensions.Logging.Testing;
 using Xunit;
 
 namespace Microsoft.Extensions.Logging
```

Comment:
A few of these test classes should be in the `M.E.Logging.Test` namespace instead of `M.E.Logging`